api/spec/requests/spree/api/v2/storefront/account/addresses_spec.rb

```diff
@@ -67,6 +67,24 @@
       end
     end
 
+    context 'when address from countries that are supported in the current store' do
+      let(:current_store) { Spree::Store.first }
```

Comment:
```suggestion
```

Reply:
@damianlegawiec we don't have store variable here, so I moved it to the top of the file